lib/applications/validation.ts

```diff
@@ -59,6 +59,13 @@ export const editAdditionalQuestionsSchema = object({
   additionalInformation: additionalQuestionsSchema,
 });
 
+/**
+ * Validation schema for additional information mutation
+ */
+export const additionalQuestionsMutationSchema = additionalQuestionsSchema.shape({
+  id: number().positive('Application not found').required('Application not found'),
```

Comment:
Error message here should be something like "Invalid application ID" and "Application ID missing"